packages/source-iotsitewise/src/time-series-data/client/batch.spec.ts

```diff
@@ -237,6 +237,83 @@ describe('createRawHistoricalEntryBatches', () => {
     );
   });
 
+  it.only.each(['fetchMostRecentBeforeEnd', 'fetchMostRecentBeforeStart'])(
+    'separate all %s entries',
+    (mostRecentOption) => {
+      const onError = jest.fn();
+      const onSuccess = jest.fn();
+      const startDate = new Date(2000, 0, 0);
+      const endDate = new Date(2001, 0, 0);
+      const resolution = '0';
+
+      const batches = createRawHistoricalEntryBatches([
```

Comment:
Would it be valuable to add or extend test cases to show the single data point requests are not combined with requests of other sizes? Or is this already covered in the existing tests?

Reply:
that's the intent of this test; @line#276, it expects 2 batches of requests 🙏